packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -70,16 +75,32 @@ describe("Peer Exchange", () => {
       expect(peerInfos.length).to.be.greaterThan(0);
       expect(peerInfos.length).to.be.lessThanOrEqual(numPeersToRequest);
       expect(peerInfos[0].ENR).to.not.be.null;
+      expect(peerInfos[0].ENR?.peerInfo?.multiaddrs).to.not.be.null;
 
+      let foundNodeMas: Multiaddr[] = [];
+      let foundNodePeerId: PeerId | undefined = undefined;
       const doesPeerIdExistInResponse =
-        peerInfos.find(
-          ({ ENR }) => ENR?.peerInfo?.id.toString() === nwaku1PeerId.toString()
-        ) !== undefined;
+        peerInfos.find(({ ENR }) => {
```

Comment:
I would not use `.find` in that case and would use `.forEach` since side effects important so that presence of `foundNodePeerId` is the result.

Reply:
changed it to use `some` which seems more appropriate & efficient